app/views/collections/stock-sales.md

```diff
@@ -9,3 +9,26 @@
 [WGBH Stock Sales](http://wgbhstocksales.org/) is the clip licensing division of the WGBH Media Library and Archives. Stock Sales licenses originally shot WGBH footage and web content to television producers, publishers and professional media organizations. The Stock Sales website features over 10,000 clips and 12 WGBH collections that users can search, browse, and license.
 
 Search and browse thousands of WGBH clips available to license! Click on the thumbnails below to screen examples:
+
+## Examples
+
+<div class="document col-md-4 col-sm-6">
+    <a href="http://wgbhstocksales.org/catalog/TODO">
+        <img src="TODO">
+        <div class="info">Title goes here</div>
+    </a>
+</div>
+
+<div class="document col-md-4 col-sm-6">
+    <a href="http://wgbhstocksales.org/catalog/TODO">
+        <img src="TODO">
+        <div class="info">Title goes here</div>
+    </a>
+</div>
+
+<div class="document col-md-4 col-sm-6">
+    <a href="http://wgbhstocksales.org/catalog/TODO">
+        <img src="TODO">
+        <div class="info">Title goes here</div>
+    </a>
+</div>
```

Comment:
@mccalluc i'm not crazy about this. We're introducing Bootstrap's responsive grid system logic into our content database.

These are the tabs, right? What constitutes a tab? Is it just an image and a title?